x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -173,3 +173,946 @@ host:keyword | semantic_text_field:semantic_text
 "host2"      | all we have to decide is what to do with the time that is given to us
 "host3"      | be excellent to each other
 ;
+
+case
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = case(st_ip == "1.1.1.1", "okay", "try again")
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:keyword
+1           | okay
+2           | try again
+3           | try again
+;
+
+coalesce
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = coalesce(st_version, st_ip, semantic_text_field)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:semantic_text
+1           | 1.2.3
+2           | 9.0.0
+3           | be excellent to each other
+;
+
+greatest
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = greatest(semantic_text_field, st_version)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:semantic_text
+1           | live long and prosper
+2           | all we have to decide is what to do with the time that is given to us
+3           | null
+;
+
+least
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = least(semantic_text_field, st_version)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:semantic_text
+1           | 1.2.3
+2           | 9.0.0
+3           | null
+;
+
+convertToBool
```

Comment:
tests for [Type conversion functions](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-type-conversion-functions)